libs/wire-api/src/Wire/API/User/Handle.hs

```diff
@@ -42,6 +44,7 @@ import Wire.API.Arbitrary (Arbitrary, GenericUniform (..))
 newtype UserHandleInfo = UserHandleInfo {userHandleId :: UserId}
```

Comment:
this data type needs to contain a domain as well.

Reply:
Yes, I also discovered that we missed it while creating new endpoints to get User, I will fix those too.